nvme.c

```diff
@@ -1893,6 +1896,204 @@ static int get_supp_cap_config_log(int argc, char **argv, struct command *cmd,
 	return err;
 }
 
+static int io_mgmt_send(int argc, char **argv, struct command *cmd, struct plugin *plugin)
+{
+	const char *desc = "I/O Management Send";
+	const char *data = "optional file for data (default stdin)";
+	const char *data_len = "length of data in bytes";
```

Comment:
Seems this is also can be moved at the beginning of the file as common strings. By the way there is also the folloing common string already so it also can be used instead.
```
static const char *buf_len = "buffer len (if) data is returned";
```

Reply:
Switch to `buf_len`, but updated the description a little bit since it is used for both sends and receives.